ledger/heavy/executor/jetkeeper.go

```diff
@@ -20,143 +20,358 @@ import (
 	"context"
 	"sync"
 
+	"github.com/insolar/insolar/insolar/pulse"
+	"github.com/insolar/insolar/instrumentation/inslogger"
+	"github.com/insolar/insolar/network/storage"
 	"github.com/pkg/errors"
 
 	"github.com/insolar/insolar/insolar"
 	"github.com/insolar/insolar/insolar/jet"
-	"github.com/insolar/insolar/instrumentation/inslogger"
 	"github.com/insolar/insolar/internal/ledger/store"
 )
 
 //go:generate minimock -i github.com/insolar/insolar/ledger/heavy/executor.JetKeeper -o ./ -s _gen_mock.go
 
 // JetKeeper provides a method for adding jet to storage, checking pulse completion and getting access to highest synced pulse.
 type JetKeeper interface {
-	// Add performs adding jet to storage and checks pulse completion.
-	Add(context.Context, insolar.PulseNumber, insolar.JetID) error
+	// AddDropConfirmation performs adding jet to storage and checks pulse completion.
+	AddDropConfirmation(context.Context, insolar.PulseNumber, insolar.JetID, bool) error
```

Comment:
Let's give names to parameters for clarification. `bool` ar the end isn't obvious.